index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Abel Seifu</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header class="header">
+        <a href="#headline-section" class="header-logo">My Logo</a>
+        <nav>
+          <button class="hamburger-btn">
+            <img src="Union.svg" alt="Hamburger Navbar Icon" />
+          </button>
+          
+        </nav>
+      </header>
+     
+      <main class="main">
+        <section class="headline" id="headline-section">
+          <h1>
+            I’m Abel <br />
+            Glad to see you!
+          </h1>
+  
+          <div class="headline-text">
+            <p>
+              I am a software developer! I can help you build a product, feature
+              or website. Look through some of my work and experience! If you like
+              what you see and have a project you need coded, do not hestiate to
+              contact me.
+            </p>
+          </div>
+          
+          <div>
+            <section class="connect">
+                <h2>Let's Connect</h2>
+            </section>
```

Comment:
- [ ] As requested by the previous reviewer Kindly make sure your main title `h1` and the `<h2>Let's Connect</h2>` has the same `font size` and `font weight` as seen in the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). 🙏 

- For the main title `h1` :
```
font-size: 40px;
font-weight: 700;
line-height: 52px;
```

- For the <h2>Let's Connect</h2>:
```
font-size: 16px;
font-weight: 500;
line-height: 24px;
```
Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/79058364/185624608-70505bc7-ab9c-4c42-a98d-2b2c808c5873.png) | ![18](https://user-images.githubusercontent.com/79058364/185624419-e4ab8bd0-f89d-41da-bc0b-f123e55d1f34.png)


_Originally posted by @Ugboaja-Uchechi in https://github.com/Abe1able/Personal-Portfolio/pull/1#discussion_r949715031_